source/content.js

```diff
@@ -73,7 +72,6 @@ import addScopedSearchOnUserProfile from './features/add-scoped-search-on-user-p
 import monospaceTextareas from './features/monospace-textareas';
 import improveShortcutHelp from './features/improve-shortcut-help';
 import displayIssueSuggestions from './features/display-issue-suggestions';
-import addPullRequestHotkey from './features/add-pull-request-hotkey';
```

Comment:
Did you mean to remove this?

Reply:
Good catch. I didn’t. It was a bad manual merge